controller/ovn-controller.c

```diff
@@ -2562,6 +2564,18 @@ check_northd_version(struct ovsdb_idl *ovs_idl, struct ovsdb_idl *ovnsb_idl,
     return true;
 }
 
+static bool print_records_callback(uintptr_t arg, const ddlog_record *rec, ssize_t weight) {
```

Comment:
The OVS/OVN style calls for the function name to start on its own line, and the `{` to be on its own line too, like this:

```
static bool
print_records_callback(uintptr_t arg, const ddlog_record *rec, ssize_t weight)
{
```